spec/site_prism/waiter_spec.rb

```diff
@@ -26,5 +26,22 @@
 
       expect(duration).to be_within(0.1).of(timeout)
     end
+
+    context 'when time is frozen' do
+      before do
```

Comment:
this could probably be simplified to `allow(Time).to receive(:now).and_return { Time.new(2019,4,25) }` to save you having to open the class up - Might be a bit shorter as well